### PR TITLE
update prepare params and tests for only text or html

### PR DESCRIPTION
**Pull request by ros3bud**
Takes care of #10.

@alexkwolfe, LGTY?

### CODE REVIEW

File: spec/send_spec.coffee [hunk 1]
```diff
@@ -137,6 +137,36 @@ describe 'successfully send email', ->
...
+    scope = nock('https://email.us-east-1.amazonaws.com:443')
+      .post('/', "Action=SendEmail&Destination.BccAddresses.member.1=cheese%40eat.com&Destination.CcAddresses.member.1=lynda%40activeprospect.com&Destination.ToAddresses.member.1=lynda%40activeprospect.com&Message.Body.Text.Data=Best%20new%20chefs&Message.Subject.Data=Food%20and%20Wine&Source=no-reply%40offerconfirmation.com&Version=2010-12-01")
+      .reply(200, "<SendEmailResponse xmlns=\"http://ses.amazonaws.com/doc/2010-12-01/\">\n  <SendEmailResult>\n    <MessageId>0000014e979dba43-e931c8fa-047e-47c2-8a62-ce49e6223000-000000</MessageId>\n  </SendEmailResult>\n  <ResponseMetadata>\n    <RequestId>948f2c25-2bd4-11e5-878e-172e34e217cf</RequestId>\n  </ResponseMetadata>\n</SendEmailResponse>\n", 
+      { 
+        'x-amzn-requestid': '948f2c25-2bd4-11e5-878e-172e34e217cf',
+        'content-type': 'text/xml',
+        'content-length': '326',
+        'date': 'Thu, 16 Jul 2015 16:06:13 GMT' 
+      }
+    )
```
Comment (alexkwolfe) [inline spec/send_spec.coffee]: Formatting is messed up here.

File: src/send.coffee [hunk 2]
```diff
@@ -74,16 +74,16 @@ prepareParams = (vars) ->
...
-        Html:
-          Data: vars.html
-        Text: 
-          Data: vars.text
+      Body: {}
 
-  params.Destination.CcAddresses = [vars.cc.valueOf()] if vars.cc
+  params.Message.Body.Html  =  Data: vars.html if vars.html
+  params.Message.Body.Text  =  Data: vars.text if vars.text
+  delete params.Message.Body if !vars.html and !vars.text
```
Comment (alexkwolfe) [inline src/send.coffee]: Too much whitespace around the `=`

Comment (alexkwolfe) [inline src/send.coffee]: This is a minor thing, but it's sort of odd how you add the `Body` and then go back to delete it later. Why not skip those machinations and only add the body if the text or html is present?

**Comment by alexkwolfe** [comment]
Other than those issues, LGTM

File: src/send.coffee (hunk 2 above)
Comment (ros3bud) [inline src/send.coffee]: Good point, I see what you mean there.

**Pull request merged**